packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CheckoutConfirm/index.tsx

```diff
@@ -27,6 +27,15 @@ class CheckoutConfirm extends PureComponent<Props> {
       this.props.buySellActions.fetchSDDVerified()
       this.props.buySellActions.fetchCards(false)
       this.props.brokerageActions.fetchBankTransferAccounts()
+      if (this.props.pendingOrder) {
```

Comment:
So for the buy flow we actually get the deposit terms from the quote endpoint so we don't want to fetch the deposit terms again here.